node/src/board.js

```diff
@@ -74,5 +87,40 @@ let boardModule = {
         }
         return board;
     },
+
+
+    ligneASupprimer: function(board){
```

Comment:
Why is this function in an object? Does it bring any value?

Reply:
If it does, put it in a class at least, and write it this way: `deleteCardLine: (board) => { /*...*/ }`